### PR TITLE
Updated user documentation and user guide links for SHU BMRC

### PR DESCRIPTION
**Pull request by lquayle88**
---
name: SHU BMRC
about: Updated config and user documentation
---

Please follow these steps before submitting your PR:

- [X] If your PR is a work in progress, include `[WIP]` in its title
- [X] Your PR targets the `master` branch
- [X] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [X] Add your custom config file to the `conf/` directory
- [X] Add your documentation file to the `docs/` directory
- [X] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [X] Add your custom profile to the `README.md` file in the top-level directory
- [X] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
- [X] OPTIONAL: Add your custom profile path and GitHub user name to `.github/CODEOWNERS` (`**/<custom-profile>** @<github-username>`)

### CODE REVIEW

File: docs/shu_bmrc.md [hunk 1]
```diff
@@ -1,13 +1,88 @@
...
+- **Location**: BMRC HPC Cluster at Sheffield Hallam University
+- **Contact**: Dr Lewis A Quayle ([l.quayle@shu.ac.uk](mailto:l.quayle@shu.ac.uk))
+- **Documentation**: [BMRC HPC Documentation](https://bmrc-hpc-documentation.readthedocs.io/en/latest/)
+
+## Requirements
+
+To use this configuration, you must have:
+
+- **Access to BMRC HPC**: Ensure your user account is enabled for HPC access at Sheffield Hallam University. The **GlobalProtect VPN** is required for remote access. For setup instructions, refer to [SHU VPN Guide](https://www.shu.ac.uk/digital-skills/programs-and-applications/virtual-private-network-vpn).
+- **Nextflow**: Version 20.10.0 or later is recommended for optimal compatibility.
```
Comment (jfy133) [inline docs/shu_bmrc.md:33]: That's extremely old, are you still using DSL1 pipelines? If so I would recommend to use 22.10.6 as the minimum version (the last DSL1 supporting version of Nextflow)

File: docs/shu_bmrc.md [hunk 2]
```diff
@@ -1,13 +1,88 @@
...
+| Memory   | 1007 GB   |
+| Time     | 999 hours |
+
+### Container Support
+
+The configuration supports Apptainer for containerised workflows, with automatic mounting enabled, allowing seamless access to necessary filesystems within containers.
+
+### Cleanup
+
+Intermediate files from successful runs will be automatically deleted to free up storage.
```
Comment (jfy133) [inline docs/shu_bmrc.md:64]: You can _maybe_ mention if you need to debug a run without cleaning up  by including the `debug` profile, if you trust your users :grimacing:

**Review by jfy133 — APPROVED** [review]

File: docs/shu_bmrc.md (hunk 1 above)
Comment (lquayle88) [reply docs/shu_bmrc.md:33]: Thanks James. This was carryover from an old guide I wrote (another win for copy-paste). I will update accordingly.

File: docs/shu_bmrc.md (hunk 2 above)
Comment (lquayle88) [reply docs/shu_bmrc.md:64]: Most of them are new HPC users, so I'll maybe leave this as is for the time being

**Review by jfy133 — APPROVED** [review]

**Pull request merged**